.gitignore

```diff
@@ -1 +1,3 @@
-/target
+target/
+build/
```

Comment:
You require a minimum of Meson 0.59.0, so you are covered by this guarantee: https://mesonbuild.com/FAQ.html#how-do-i-ignore-the-build-directory-in-my-vcs

You therefore don't need to add your own ignore for the build directory -- which anyways could be called `builddir/` or `builddir-release/` or `builddir-debug/` or `.build/` or `_build/` or any of a variety of other common naming styles.